### ISSUE TITLE
Incorrect timestamp when submitting to pskreporter

### ISSUE THREAD

**Issue by pjsg** [alexranaldi/CWSL_DIGI#7]
It looks as though CWSL_DIGI submits the timestamp of the *start* of the transmission rather than the *decode* time. This confuses the analyzer that looks for incorrect QRG on the reports -- it assumes that a particular callsign only transmits on a single band in a single timeslot. 

Thanks

Philip, N1DQ (PskReporter.info)

**Comment by Supremehamster** [comment]
Won't the decode time vary though depending on peoples cores?

**Comment by pjsg** [comment]
Yes it does (a bit). These are some counts of which second was reported for FT8:

```
2918875 0
 423131 1
 192628 2
  99906 3
  55023 4
  31087 5
  18745 6
  13178 7
  10655 8
   8843 9
  17135 10
4653483 11
6941843 12
 886087 13
4444038 14
```

I expect that some of the outliers are clock errors on the system in question.

**Comment by alexranaldi** [comment]
I thought the convention was to use the start time.  WSJT-X timestamps decoded messages according to the period in which they begin. Right now I'm just passing along the WSJT-X timestamps. I can change it to whatever works best for PSK Reporter, if desired.
